package.json

```diff
@@ -45,6 +45,6 @@
     "isolated-vm": "^5.0.1"
   },
   "peerDependencies": {
-    "@wonderlandengine/api": "^1.3.0"
```

Comment:
same here. i think this was a mistake before as well; this field should have been using version ranges